routers/web/web.go

```diff
@@ -872,7 +872,7 @@ func RegisterRoutes(m *web.Route) {
 						m.Delete("", repo.DeleteProjectBoard)
 						m.Post("/default", repo.SetDefaultProjectBoard)
 
-						m.Post("/{index}", repo.MoveIssueAcrossBoards)
+						m.Post("/{issueID}/{sorting}", repo.MoveIssueAcrossBoards)
```

Comment:
I think `issueIndex` is better. This concept is used everywhere. `issueID` is misleading.

Reply:
But maybe we should switch to use issue index but not id here.